nelmio/api-doc-bundle/2.13/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+    "bundles": {
+        "Nelmio\\ApiDocBundle\\NelmioApiDocBundle": ["all"]
```

Comment:
Is it common to have ApiDoc available on prod too?

I have never used it manually, but on projects I used to work on in the past, NelmioApiDocBundle was enabled only in dev mode 🤔